catalyst/utils/config.py

```diff
@@ -254,6 +254,13 @@ def parse_config_args(*, config, args, unknown_args):
                 continue
             config["args"][key] = value
 
+    if safitty.get(config, "args", "autoresume", default=False) and \
+            safitty.get(config, "args", "logdir") is not None and \
+            safitty.get(config, "args", "resume") is None:
+        log_dir = Path(safitty.get(config, "args", "logdir"))
+        checkpoint_filename = log_dir / "checkpoints" / "last_full.pth"
+        if checkpoint_filename.is_file():
```

Comment:
btw, does this function also checks, that file exists?

Reply:
From docs:
Path.is_file()
Return True if the path points to a regular file (or a symbolic link pointing to a regular file), False if it points to another kind of file.

False is also returned if the path doesn’t exist or is a broken symlink; other errors (such as permission errors) are propagated.